engine/src/main/coffee/engine/workspace.coffee

```diff
@@ -170,4 +170,5 @@ module.exports =
       updater
       userDialogPrims
       world
+      i18nBundle
```

Comment:
I needed to exposure the i18nBundle so that TU and potentially Galapagos could call it and switch the current language based on the client or the browser's preference.